### PR TITLE
Setting (Frozen)Time::defaultOutputTimezone docs

### PR DESCRIPTION
**Pull request by ionas**

### CODE REVIEW

File: en/core-libraries/time.rst [hunk 1]
```diff
@@ -236,6 +236,40 @@ Likewise, it is possible to alter the default formatting string to be used for
 It is recommended to always use the constants instead of directly passing a date
 format string.
 
+.. versionadded:: 3.2.11
+
+You can change the default timezone used for outputting your whole app in `app.php`:
```
Comment (ravage84) [inline en/core-libraries/time.rst]: app -> application

File: en/core-libraries/time.rst [hunk 2]
```diff
@@ -236,6 +236,40 @@ Likewise, it is possible to alter the default formatting string to be used for
...
+
+    'defaultOutputTimezone' => 'Europe/Berlin',
+
+... or overwrite fetching from app config in `bootstrap.php`:
```
Comment (ravage84) [inline en/core-libraries/time.rst]: dito

File: en/core-libraries/time.rst [hunk 3]
```diff
@@ -236,6 +236,40 @@ Likewise, it is possible to alter the default formatting string to be used for
...
+
+    // ...
+
+    public function initialize() {
+        parent::initialize();
+        Time::setDefaultOutputTimezone('America/Vancouver');
+        FrozenTime::setDefaultOutputTimezone('America/Vancouver');
+    }
+
+In a similar way you will be able to set a goip, browser-based, profile or session based
```
Comment (ravage84) [inline en/core-libraries/time.rst]: What's "goip"?

File: en/core-libraries/time.rst [hunk 4]
```diff
@@ -236,6 +236,40 @@ Likewise, it is possible to alter the default formatting string to be used for
...
+default output timezone, for instance by calling:
```
Comment (ravage84) [inline en/core-libraries/time.rst]: double colon

File: en/core-libraries/time.rst [hunk 5]
```diff
@@ -236,6 +236,40 @@ Likewise, it is possible to alter the default formatting string to be used for
...
+default output timzeone by calling `setDefaultOutputTimezone()`, for example:
+
+    use Cake\I18n\FrozenTime;
+    use Cake\I18n\Time;
+
+    // ...
+
+    public function initialize() {
+        parent::initialize();
+        Time::setDefaultOutputTimezone('America/Vancouver');
```
Comment (ravage84) [inline en/core-libraries/time.rst]: You could save the Timezone string into a variable and use that one, to better showcase using it for both.

File: en/core-libraries/time.rst [hunk 6]
```diff
@@ -236,6 +236,40 @@ Likewise, it is possible to alter the default formatting string to be used for
...
+        FrozenTime::setDefaultOutputTimezone('America/Vancouver');
+    }
+
+In a similar way you will be able to set a goip, browser-based, profile or session based
+default output timezone, for instance by calling:
+
+    public function beforeFilter() {
+        parent::beforeFilter();
+        Time::setDefaultOutputTimezone($this->request->session->read('defaultUserTimezone'));
```
Comment (ravage84) [inline en/core-libraries/time.rst]: Same as above.

File: en/core-libraries/time.rst (hunk 3 above)
Comment (ionas) [inline en/core-libraries/time.rst]: GeoIP

**Comment by ionas** [comment]
Thanks for the feedback. Hope its clear now.

File: en/core-libraries/time.rst [hunk 7]
```diff
@@ -236,6 +236,44 @@ Likewise, it is possible to alter the default formatting string to be used for
...
 format string.
 
+.. versionadded:: 3.2.11
+
+You can change the default timezone used for outputting your whole application
+in `app.php`::
+
+    'defaultOutputTimezone' => 'Europe/Berlin',
+
+... or overwrite fetching from application config in `bootstrap.php`::
```
Comment (markstory) [inline en/core-libraries/time.rst]: You need 2 backticks to get inline code in sphinx.

File: en/core-libraries/time.rst [hunk 8]
```diff
@@ -236,6 +236,44 @@ Likewise, it is possible to alter the default formatting string to be used for
...
+
+    $timezone = 'Europe/Berlin';
+    Time::setDefaultOutputTimezone($timezone);
+    FrozenTime::setDefaultOutputTimezone($timezone);
+
+At any point in your application, say in a `FrontendAppController` you may change the
+default output timzeone by calling `setDefaultOutputTimezone()`, for example::
```
Comment (markstory) [inline en/core-libraries/time.rst]: Need to double all the backticks here.

File: en/core-libraries/time.rst (hunk 7 above)
Comment (antograssiot) [inline en/core-libraries/time.rst]: filename where supposed to be in bold so surrounded by **

Comment (jhli) [inline en/core-libraries/time.rst]: So ** it will be without backticks

Comment (antograssiot) [inline en/core-libraries/time.rst]: @ionas yeah

**Comment by ionas** [comment]
I hope its good to merge now.

**Comment by ionas** [comment]
Should I add some more docs explaining why DateTime works but Date does not?
Ref: https://github.com/cakephp/cakephp/issues/8948

**Comment by antograssiot** [comment]
👍

**Comment by cake17** [comment]
@ionas nice add and I'm 👍  for more docs about Date/DateTime

**Comment by ionas** [comment]
This feature has been merged but then removed. So the docs are not required anymore.
Sorry.

**Pull request closed without merging**